.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,32 +1,50 @@
-# EASI-0000, EASI-0000
-<!-- Follow the pattern of Parent issue, Child issue, if appropriate -->
-
-## Changes and Description
-
-- Change 1
-- Change 2
+<!--
+REQUIRED
+    Ensure that your PR title has the relevant Jira ticket number(s) in the title.
+    Follow this pattern: [EASI-1234] [EASI-4567] Title of the PR.
+    Use [NOREF] in place of a ticket number when there's no associated Jira ticket.
+    The heading below should just have the ticket numbers (for easy linking in Jira)
+-->
+# EASI-0000
+
+## Description
+<!--
+REQUIRED
+    Provide details as to what the PR aims to accomplish
+    Be as descriptive as you can, and include any relevant information that will help the reviewer understand the scope of the changes
+    Include screenshots or screen recordings to assist in reviewing if possible.
+-->
 
-<!-- Put a description here! -->
 
 ## How to test this change
-
 <!--
-    Add any steps or code to run in this section to help others run your code:
-
-    ```sh
-    echo "Code goes here"
-    ```
---->
-
-## PR Author Review Checklist
+REQUIRED
+    Add instructions on how to test the changes in this PR
+    This can be a list of steps to reproduce a bug, or a list of steps to verify a feature in the application
+    Include any example shell commands, SQL queries/commands, or Postman requests that reviewers can run to test the changes
+-->
+- 
+
+## PR Author Checklist
+<!--
+REQUIRED
+    Ensure that each of the following is true before you submit this PR (or before it leaves "draft" status), and check each box to confirm
+-->
 
-- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
-- [ ] Added or updated tests for backend resolvers or other functions as needed.
-- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
+- [ ] I have provided a detailed description of the changes in this PR.
+- [ ] I have provided clear instructions on how to test the changes in this PR.
+- [ ] I have updated tests or written new tests as appropriate in this PR.
 - [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.
```

Comment:
This to me is the most useful change in this PR. I feel like it is contextually sensible.